wp-content/plugins/automatic-translation/TranslationManager.php

```diff
@@ -121,6 +139,15 @@ private function create_translation($post, $source_language_code, $target_langua
 		$this->add_save_post_hook();
 		$this->wpml_helper->link_wpml($post->ID, $post->post_type, $translated_post_id, $target_language_code);
 		$this->mark_as_automatic_translation($translated_post_id);
+		
+		//fixes wrong post_parent for autotranslation
```

Comment:
I found earlier, that you can fix the problem where a page is not belonging to its parent by clicking on publish again. And doing the update also works. I cannot say if this is a bug in WPML, Wordpress or our plugin.